service/src/main/java/bio/terra/pipelines/App.java

```diff
@@ -36,12 +39,6 @@ public static void main(String[] args) {
     new SpringApplicationBuilder(App.class).initializers(new LoggingInitializer()).run(args);
   }
 
-  private final DataSource dataSource;
-
-  public App(DataSource dataSource) {
```

Comment:
yeah, couldn't find any usage and the service seems to run fine without it. I think this might have been a holdover from TPS, which has a more complicated database setup than we need